docs/develop/guides/working-with-momento-tokens.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position: 4
+sidebar_label: Momento Authentication
+title: Momento Authentication With Expiring Tokens
+description: Learn how to use expiring tokens to enhance the security of your application
+pagination_next: null
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Momento authentication with expiring tokens
+
+To access Momento services from your application, a Momento auth token is required. The auth token is an alphanumeric string Momento generates that is unique to your account and the cloud provider's Region you select. When creating an auth token, you can set up one of two types of expiration:
+
+* An auth token that never expires. This is used for dev/test deployments.
+* An auth token which expires and needs periodic rotation. You can set the expiration to 7, 30, 60, 90 days or custom, where you can select a specific calendar date. An expiring auth token is considered best practice for production deployments. We will walk through how to do exactly that in this article.
+
+:::info
+In addition to using an expiring auth token, it is best practice to store an auth token in a service like [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/) or [GCP Secret Manager](https://cloud.google.com/secret-manager). Go to [Retrieving a Momento auth token from AWS Secrets Manager](/develop/sdks-integrations/aws-secrets-manager) for an example on using this.
```

Comment:
Do we want to update examples in that to include what you would store for an expiring token?

Reply:
Once the prepackaged Lamda function app has determined what should go in there, I plan to update the page linked in the section you highlighted.